release/apparmor.go

```diff
@@ -292,43 +315,43 @@ func assessAppArmor() {
 	appArmorSummary = "apparmor is enabled and all features are available"
 }
 
-func probeAppArmorKernelFeatures() []string {
+func probeAppArmorKernelFeatures() ([]string, error) {
 	// note that ioutil.ReadDir() is already sorted
 	dentries, err := ioutil.ReadDir(appArmorFeaturesSysPath)
 	if err != nil {
-		return []string{}
+		return []string{}, err
 	}
 	features := make([]string, 0, len(dentries))
 	for _, fi := range dentries {
 		if fi.IsDir() {
 			features = append(features, fi.Name())
 		}
 	}
-	return features
+	return features, nil
 }
 
-func probeAppArmorParserFeatures() []string {
-	parser := findAppArmorParser()
-	if parser == "" {
-		return []string{}
+func probeAppArmorParserFeatures() ([]string, error) {
+	parser, err := findAppArmorParser()
+	if os.IsNotExist(err) {
```

Comment:
If findAppArmorParser() is updated to return something other than IsNotExist, then we won't return an error. Why not just check if there is an error?

Reply:
Fixed, thank you for being thorough :-)